x-pack/test/functional/apps/spaces/spaces_selection.ts

```diff
@@ -27,16 +27,70 @@ export default function spaceSelectorFunctonalTests({ getService, getPageObjects
         });
 
         await PageObjects.spaceSelector.clickSpaceCard(spaceId);
-
         await PageObjects.spaceSelector.expectHomePage(spaceId);
 
-        await PageObjects.spaceSelector.openSpacesNav();
-
         // change spaces
+        await PageObjects.spaceSelector.switchToSpace('default');
+        await PageObjects.spaceSelector.expectHomePage('default');
+      });
+    });
 
-        await PageObjects.spaceSelector.clickSpaceAvatar('default');
+    describe('Spaces Data', () => {
+      const spaceId = 'another-space';
+      const paths = {
+        dashboard: '/dashboard',
+        sampleData: '/home/tutorial_directory/sampleData',
+      };
+      const expectDashboardRenders = async (dashName: string) => {
+        await PageObjects.dashboard.searchForDashboardWithName(dashName);
+        await PageObjects.dashboard.selectDashboard(dashName);
+        await PageObjects.header.waitUntilLoadingHasFinished();
+        await PageObjects.dashboard.waitForRenderComplete(); // throws if all items are not rendered
+      };
 
-        await PageObjects.spaceSelector.expectHomePage('default');
+      before(async () => {
+        await esArchiver.load('spaces');
+        await PageObjects.security.login(null, null, {
+          expectSpaceSelector: true,
+        });
+        await PageObjects.spaceSelector.clickSpaceCard('default');
+        await PageObjects.spaceSelector.switchToSpace('default', {
+          redirectPath: paths.sampleData,
+        });
+        await PageObjects.home.addSampleDataSet('logs');
+        await PageObjects.spaceSelector.switchToSpace(spaceId, {
+          redirectPath: paths.sampleData,
+        });
+        await PageObjects.home.addSampleDataSet('flights');
+      });
+
+      after(async () => {
+        await PageObjects.spaceSelector.switchToSpace('default', {
+          redirectPath: paths.sampleData,
+        });
+        await PageObjects.home.removeSampleDataSet('logs');
+        await PageObjects.spaceSelector.switchToSpace(spaceId, {
+          redirectPath: paths.sampleData,
+        });
+        await PageObjects.home.removeSampleDataSet('flights');
```

Comment:
TBH I'm not sure if removing the sample data is a necessary cleanup step for this test or not -- added it just to be safe.